src/Symfony/Component/Console/Command/Command.php

```diff
@@ -420,9 +420,9 @@ public function addArgument(string $name, int $mode = null, string $description
     /**
      * Adds an option.
```

Comment:
overall, we got curious if phpdoc_align make sense on Fixer side, especially because of such cases like here. WDYT?
The version proposed by PR is one in-line with current CS rules.

Reply:
I agree it might be better to not align those

Reply:
I would love to have conclusion that "always align" or "never align".

If we say always, let's pay this little price of having this single case of 100k cases ugly. Maintaining exception and not letting tool to do it's job is painful and preventing auto-apply of tool results.

Reply:
I don't think the current changes on this are improving the source. Consistency is nice, but this goes against readability here to me.
I understand you'd like a rule, so maybe we could just decide which length is too much to align? or say that complex types shouldn't trigger alignment?
we can revert these from the PR and discuss that separately if you don't want to other changes to wait for this to be resolved

Reply:
for 6k files in this repo, we have the rule not applied for __TWO__ cases only. instead of making some special handling of this and implementing rule for exception, I would ask to get this applied. Let's be pragmatic here and avoid increasing rule complexity for such rare case.

Reply:
we accept exceptions so it's fine have two cases that don't follow the rule
please revert the related changes in this PR and :rocket:

Reply:
It makes it tricky to apply the CS (and on long run, have CS always green) - because manually reverting the exceptions each time :(

reverting as requested